static/jade/pages/map.jade

```diff
@@ -1,10 +1,9 @@
 h2 The venue
 
 address
-  p.strong VUT FIT
-  p Building D
-  p Božetěchova 1/2
-  p 612 66 Brno
+  p.strong Masaryk University, Faculty of Informatics
+  p Botanická 68a
```

Comment:
I wouldn't change address unless we're 100% sure. I know that MU FI is winning option at the moment, but let's wait a while until is confirmed. We don't want to change venue info every week :)